collector/aws/resources/rds.go

```diff
@@ -197,6 +197,10 @@ func (r *RDSManager) getPricingFilterInput(instance *rds.DBInstance) pricing.Get
 		databaseEngine = "PostgreSQL"
 	case "aurora":
 		databaseEngine = "Aurora MySQL"
+	case "aurora-mysql":
```

Comment:
you can merge the case: `aurora-mysql` and `aurora-postgresql` in one case with comma